### ISSUE TITLE
What is this repo vs DataBiosphere/terra-docker? 

### ISSUE THREAD

**Issue by nturaga** [anvilproject/anvil-docker#1]

**Comment by rtitle** [comment]
Hey @nturaga,

In the short term this repo will be used for storing the RStudio CE image to be launched by Leo. In the future we plan to put a RStudio Pro image in the `terra-docker` repo and make that fully supported by Terra. But license negotiations are not done yet and we don't want to make RStudio CE a supported part of Terra because of other collaborators' concerns about software licenses. So the workaround from a legal perspective is to treat RStudio CE as an AnVIL-provided image with its own repo and GCR.

TLDR: it is a temporary repo to get RStudio CE in production more quickly; with the end state being to have RStudio Pro in `terra-docker`.

-Rob
